spyder/app/utils.py

```diff
@@ -145,8 +142,8 @@ def filter(self, record):
             root_logger.addHandler(handler)
 
 
-def delete_lsp_log_files():
-    """Delete previous dead Spyder instances LSP log files."""
+def delete_debug_log_files():
+    """Delete previous dead Spyder instances LSP and debug log files."""
```

Comment:
```suggestion
    """Delete previous debug log files."""
```

I think the previous docstring was a bit hard to understand.